src/main/java/frc/robot/commands/ReleaseAlgae.java

```diff
@@ -3,8 +3,11 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.AlgaeGripper;
 
+import java.time.Instant;
+
 public class ReleaseAlgae extends Command {
     private final AlgaeGripper algaeGripper;
+    Instant previousTime = null;
```

Comment:
don't use `Instant` as it uses the less trusty system clock. Use `Timer.getFPGATimestamp()` instead.